### PR TITLE
Adding T1087.002 Test - Ldapdomaindump on Linux

### PR DESCRIPTION
**Pull request by Leomon5**
**Details:**
This test utilizes the ldapdomaindump Python module to enumerate domain accounts and computers. 
Reference: hxxps://securityonline[.]info/ldapdomaindump-active-directory-information-dumper-via-ldap/

**Testing:**
Tested using Kali and Ubuntu against Windows Server 2019

### CODE REVIEW

File: atomics/T1087.002/T1087.002.yaml [hunk 1]
```diff
@@ -425,3 +425,50 @@ atomic_tests:
...
+    prereq_command: if [ -x "$(command -v python3 --version)" ]; then exit 0; else exit 1; fi;
+    get_prereq_command: sudo apt-get -y install python3
+  - description: Pip must be installed
+    prereq_command: if [ -x "$(command -v pip --version)" ]; then exit 0; else exit 1; fi;
+    get_prereq_command: |-
+      wget -O /tmp/get-pip.py https://bootstrap.pypa.io/pip/3.6/get-pip.py
+      sudo python3 /tmp/get-pip.py
+  - description: The ldapdomaindump module must be installed
+    prereq_command: python3 -c 'import ldapdomaindump' 2>/dev/null
+    get_prereq_command: sudo pip install ldapdomaindump
```
Comment (cyberbuff) [inline atomics/T1087.002/T1087.002.yaml:459]: `sudo` is not required to install python packages. You can change the command to 
```
pip install ldapdomaindump
```

File: atomics/T1087.002/T1087.002.yaml [hunk 2]
```diff
@@ -425,3 +425,50 @@ atomic_tests:
...
+  - description: The future module must be installed
+    prereq_command: python3 -c 'import future' 2>/dev/null
+    get_prereq_command: sudo pip install future
+  - description: The ldap3 module must be installed
+    prereq_command: python3 -c 'import ldap3' 2>/dev/null
+    get_prereq_command: sudo pip install ldap3
+  - description: The dns module must be installed
+    prereq_command: python3 -c 'import dns' 2>/dev/null
+    get_prereq_command: sudo pip install dns
```
Comment (cyberbuff) [inline atomics/T1087.002/T1087.002.yaml:468]: `ldap3` and `dnspython` will automatically get installed when installing `ldapdomaindump`. These requirements can be removed.

File: atomics/T1087.002/T1087.002.yaml [hunk 3]
```diff
@@ -425,3 +425,50 @@ atomic_tests:
...
+  dependency_executor_name: sh
+  dependencies:
+  - description: Python3 must be installed
+    prereq_command: if [ -x "$(command -v python3 --version)" ]; then exit 0; else exit 1; fi;
+    get_prereq_command: sudo apt-get -y install python3
+  - description: Pip must be installed
+    prereq_command: if [ -x "$(command -v pip --version)" ]; then exit 0; else exit 1; fi;
+    get_prereq_command: |-
+      wget -O /tmp/get-pip.py https://bootstrap.pypa.io/pip/3.6/get-pip.py
+      sudo python3 /tmp/get-pip.py
```
Comment (cyberbuff) [inline atomics/T1087.002/T1087.002.yaml:456]: `sudo` can be removed here as well.

**Review by cyberbuff — CHANGES_REQUESTED** [review]

File: atomics/T1087.002/T1087.002.yaml (hunk 3 above)
Comment (Leomon5) [reply atomics/T1087.002/T1087.002.yaml]: @cyberbuff These changes have been made. Please let me know if anything else is needed. Thank you!

**Review by cyberbuff — APPROVED** [review]

**Pull request merged**